relay-monitors/src/lib.rs

```diff
@@ -113,6 +113,10 @@ struct CheckIn {
     /// Status of this check-in. Defaults to `"unknown"`.
     status: CheckInStatus,
 
+    /// The environment to associate the check-in with
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    environment: Option<String>,
```

Comment:
Let's add some length limits to this string, like [here](https://github.com/getsentry/relay/blob/ab2a3dca7a3daaecfcdf097d898c766fd6b69476/relay-general/src/protocol/event.rs#L381). Monitor slugs may contain non-ASCII characters, if this isn't the case for `environment` consider also adding [`allow_chars`](https://github.com/getsentry/relay/blob/ab2a3dca7a3daaecfcdf097d898c766fd6b69476/relay-general/src/protocol/event.rs#L368).

Reply:
Sounds good. Thanks for linking this

Reply:
Will merge and update this

Reply:
Going to merge this as is an revisit validation